CONTRIBUTING.md

```diff
@@ -15,6 +15,18 @@ You could:
 Head over to [react-server-test-pages](/packages/react-server-test-pages) and
 check out the README to get a test server set up.
 
+## Testing
+
+Yeah!  Do it!
```

Comment:
I would move the sentence at ll 15 to after "Do it!" -- it explains how to do smoke testing.  As is, its a little imposing